src/main/features/statement-of-means/paths.ts

```diff
@@ -0,0 +1,7 @@
+import { RoutablePath } from 'common/router/routablePath'
+
+const statementOfMeansPath: string = '/case/:externalId/statement-of-means'
+
+export class Paths {
+  static readonly employmentPage = new RoutablePath(`${statementOfMeansPath}/employment`)
```

Comment:
we already had this conversation on Rafal's PR for setting this up :(, statement of means is under response which is why it doesn't have a feature setup